gemini/src/main/java/com/techempower/cache/CachedRelation.java

```diff
@@ -192,14 +192,14 @@ protected CachedRelation(EntityStore store,
     this.leftType = leftType;
     this.rightType = rightType;
     this.table = (tableName == null)
-        ? "map" + leftType.getSimpleName().toLowerCase()
+        ? leftType.getSimpleName().toLowerCase()
             + "to" + rightType.getSimpleName().toLowerCase()
         : tableName;
     this.leftColumn = (leftColumn == null)
-        ? leftType.getSimpleName().toLowerCase()
+        ? leftType.getSimpleName().toLowerCase() + "id"
         : leftColumn;
     this.rightColumn = (rightColumn == null)
-        ? rightType.getSimpleName().toLowerCase()
+        ? rightType.getSimpleName().toLowerCase() + "id"
         : rightColumn;
```

Comment:
These changes mean that in order to use default annotations (`@Relation`, `@Left`, and `@Right` without specifying table or column names) the schema for a relationship table should be:
```
footobar
-------------
fooid | barid
-------------
```